### ISSUE TITLE
Can't start fastflix on MacOS

### ISSUE THREAD

**Issue by pbakaus** [cdgriffith/FastFlix#92]
Hi there!

I've tried to install Fastflix on MacOS with your instructions, made your ffmpeg/ffprobe are available, and after installing it with pipx, I get this error:

```
fastflix-core[11540] INFO Starting FastFlix 3.2.1
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/spawn.py", line 116, in spawn_main
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/spawn.py", line 125, in _main
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/spawn.py", line 236, in prepare
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/spawn.py", line 287, in _fixup_main_from_path
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 268, in run_path
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 97, in _run_module_code
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 87, in _run_code
  File "/Users/pbakaus/.local/bin/fastflix", line 5, in <module>
  File "/Users/pbakaus/.local/pipx/venvs/fastflix/lib/python3.9/site-packages/fastflix/__main__.py", line 4, in <module>
  File "/Users/pbakaus/.local/pipx/venvs/fastflix/lib/python3.9/site-packages/fastflix/gui.py", line 91, in main
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/process.py", line 121, in start
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/context.py", line 224, in _Popen
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/context.py", line 284, in _Popen
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/popen_spawn_posix.py", line 32, in __init__
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/popen_fork.py", line 19, in __init__
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/popen_spawn_posix.py", line 42, in _launch
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/spawn.py", line 154, in get_preparation_data
  File "/usr/local/Cellar/python@3.9/3.9.0/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/spawn.py", line 134, in _check_not_importing_main
RuntimeError:
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.
```

Have you seen this before? Anything I can do on my end?

Thanks!

**Comment by cdgriffith** [comment]
Thanks for reporting that issue, I don't have a Mac so wouldn't have known! 

That is an odd one, I am trying a fix with newest version (currently in `hdr-opt` branch). 

Not sure if I can get access to a Mac this weekend to test it, so may have to go out blind.  If you're comfortable with git / command line feel free to try it yourself.

**Comment by pbakaus** [comment]
Looks like it worked! I just ran it from the `hdr-opt` branch and it works like a charm. Thanks!

**Comment by cdgriffith** [comment]
Excellent, thanks for testing it!

**Comment by cdgriffith** [comment]
Fixed in [3.3.0](https://github.com/cdgriffith/FastFlix/releases/tag/3.3.0)
